llama_index/tools/function_tool.py

```diff
@@ -2,6 +2,7 @@
 
 from pydantic import BaseModel
 from llama_index.tools.types import BaseTool, ToolMetadata
+from langchain.tools import Tool, StructuredTool
```

Comment:
a general comment:

would be nice to create a "langchain bridge" module where we route all of all langchain dependencies through. This makes it very easy to have a central place to know all the specific langchain modules we depend on for the future.

Reply:
yeah that's a good idea...there's also a bunch in `langchain_wrappers`